code/game/objects/effects/effect_system/foam.dm

```diff
@@ -51,7 +51,10 @@
 
 		flick("[icon_state]-disolve", src)
 		QDEL_IN(src, 5)
-
+	var/static/list/connections = list(
```

Comment:
what's the benefit of having a static list, rather than just doing AddElement(/datum/element/connect_loc,list(COMSIG_ATOM_ENTERED = .proc/on_cross,))?

Non blocking remark

Reply:
Otherwise it creates a new element every time